### ISSUE TITLE
Minimap: buffer scrolls up when indenting near the end of a large file.

### ISSUE THREAD

**Issue by gabrielfojo** [Komodo/KomodoEdit#900]
This only happen when the document goes beyond the height of the window, indenting more than one line together send me Upwards to page 1 -
---- now i can not reproduce it- it happen sometimes -  

Komodo Edit, version 8.5.4, build 14424, platform linux-x86_64. Built on Thu Aug

**Comment by cgchoffman** [comment]
Please try the latest Komodo Edit and see if the issue persists: http://downloads.activestate.com/Komodo/releases/9.3.1/

If it does, please provide more details.  Exactly steps taken.

- I created a PHP doc 
- added lines to the doc until it was longer than the view port
- selected multiple lines
- pressed tab
- lines indented, nothing else happened.

I can't reproduce this with the information you've provided.

**Comment by babobski** [comment]
@cgchoffman i can reproduce the problem, this bug appears when working with some larger files.
When you indent or de-indent the position is off.

Made a [screencast](https://youtu.be/8DqoDqFGk3Q) to clarify the issue.

This issue also occurs sometimes while pasting in komodo.
I'm having this bug for a long time in komodo but wasn't annoying enough for me to report.

Some more info
i don't have font-zooming enabled
Also tested it in the default skin same issue here

```[2015-11-26 18:33:59,572] [INFO] Startup: Welcome to Komodo Edit 9.3.1 build 16421 (platform win32-x86, running on Windows post2012Server version 6.3.9600)
[2015-11-26 18:33:59,572] [INFO] Startup: C:\Program Files (x86)\ActiveState Komodo Edit 9 nightly\lib\mozilla\komodo.exe built on Thu Nov 12 11:30:46 2015```

windows 10

**Comment by cgchoffman** [comment]
Ahh, it's the minimap.  

- enable minimap
- open file that is longer than the window height
- select a block of text
- hit tab

Expect: just some nice tabbing
Actual: Nice tabbing AND the buffer scrolls up

**Comment by cgchoffman** [comment]
It's #901 that I was thinking of but I don't think it's related though similar symptoms.

**Comment by babobski** [comment]
>Ahh, it's the minimap.   

Did some more testing, but it's definitely the minimap.
Had the same problems with the addons i wrote.
With the Beautify js addon when you re flow a part of code on larger files, it's getting jumpy like as in the screencast.
When i turn of the mini map this behavior disappears. Same for the Less Compiler you can compile a selection, on larger files, same issue here, and turning of the mini map resolves this behavior.

**Comment by cgchoffman** [comment]
Thanks @babobski.  I've updated the bug description to be more clear.

**Comment by babobski** [comment]
ok :+1:

**Comment by gabrielfojo** [comment]
thanks @babobski  , the issue was messing with my zen state ;)

**Comment by cdcmicro** [comment]
Wow.  I was all ready to write this up (again).  Nice the reporting work is already done.

I confirm the behavior in IDE 10 for block indent (tab), block dedent (shift+tab), and paste (shift+insert or ctrl+v).  Any chance it could get assigned/fixed sometime soon?  Maybe along with #2080 ?

Komodo IDE, version 10.1.1, build 89474, platform win32-x86.
Built on Thu Sep 01 14:27:01 2016.

**Comment by cgchoffman** [comment]
Thanks for pinging this.  I've updated the ticket.
